travis/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then     # pull request
+    bundle exec fastlane test
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" == "mPower-Android" ]]; then  # non-tag commits to master branch
+    bundle exec fastlane test
+    bundle exec fastlane assemble
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" =~ ^mPower-Android-.* ]]; then # non-tag commits to stable branches
```

Comment:
To get it to deploy to alpha we need to tag the branch, and label it "mPower-Android-.*" with the asterisk being the version or something, is that right?  I read earlier about the "stable*" branch but not seeing this reflected here